src/pages/ListScholarships.jsx

```diff
@@ -119,22 +114,34 @@ function ListScholarships() {
         <Toolbar />
 
         <Container maxWidth="md" sx={{ flexGrow: 1 }}>
-          <Box
+          <Stack
+            direction="row"
+            rowGap={2}
+            spacing={2}
+            justifyContent={isDesktop ? 'center' : 'flex-start'}
+            flexWrap={isDesktop ? 'wrap' : 'nowrap'}
             sx={{
+              mt: 2,
               overflowX: 'scroll',
-              whiteSpace: 'nowrap',
               scrollbarWidth: 'none',
+              backgroundImage:
+                'linear-gradient(to right, #F8F9FA, #F8F9FA), linear-gradient(to right, #F8F9FA, #F8F9FA), linear-gradient(to right, rgba(0, 0, 0, 0.25), rgba(255, 255, 255, 0)), linear-gradient(to left, rgba(0, 0, 0, 0.25), rgba(255, 255, 255, 0))',
+              backgroundPosition:
+                'left center, right center, left center, right center',
+              backgroundRepeat: 'no-repeat',
+              backgroundSize: '20px 100%, 20px 100%, 10px 100%, 10px 100%',
+              backgroundAttachment: 'local, local, scroll, scroll',
```

Comment:
Is this supposed to show something that hints there's more to scroll? I can't see anything on mobile nor desktop.

Reply:
yes, it shows the scroll shadow. It is hard to see since the chips are over it but you can see it in between gaps. 
<img width="520" alt="Screen Shot 2022-04-02 at 8 03 11 PM" src="https://user-images.githubusercontent.com/40483569/161409457-c1473d37-ba03-466a-b0ea-79e6b6d7b9b6.png">

Reply:
I added some padding on the top & bottom so it could be more visible 
<img width="829" alt="Screen Shot 2022-04-02 at 8 17 25 PM" src="https://user-images.githubusercontent.com/40483569/161409818-4199660a-dbd2-417d-9c57-3f2be4b9f856.png">